views/golfer.html

```diff
@@ -86,14 +86,12 @@ <h1>
         {{ $slug = print "/golfers/" .Name "/cheevos" }}
         <a {{ if ne $.Path $slug }} href="{{ $slug }}" {{ end }}>Achievements</a>
 
-        {{ $slug = print "/golfers/" .Name "/holes/" }}
-        <a {{ if not (hasPrefix $.Path $slug) }} href="{{ $slug }}bytes" {{ end }}>
-            Holes
-        </a>
-
-        {{ $slug = print "/golfers/" .Name "/rankings/" }}
-        <a {{ if not (hasPrefix $.Path $slug) }} href="{{ $slug }}bytes" {{ end }}>
-            Rankings (WIP)
+        {{ $slug = print "/golfers/" .Name }}
+        <a {{ if and (not (hasPrefix $.Path (print $slug "/rankings")))
+                    (not (hasPrefix $.Path (print $slug "/points"))) }}
+            href="{{ $slug }}/rankings/lang/bytes"
+        {{ end }}>
+            Rankings
```

Comment:
It's kinda weird that the tab always displays "Rankings" even for `/points`. This probably suggests we ought to have a common slug, I wonder if it should be `Holes` again?

Reply:
I've merged it back to a common slug Holes